invenio_pages/admin.py

```diff
@@ -20,20 +20,29 @@
 
 from __future__ import unicode_literals
 
+import os
+
 from flask import current_app
 
+from invenio.base.globals import cfg
+
 from invenio.ext.admin.views import ModelView
 from invenio.ext.sqlalchemy import db
 from invenio_pages.models import Page
 
 from jinja2 import TemplateNotFound
 
-from wtforms.validators import ValidationError
+from werkzeug import secure_filename
+from werkzeug.local import LocalProxy
+
+from wtforms.validators import DataRequired, ValidationError
 
 
 def template_exists(form, field):
     """Form validation: check that selected template exists."""
-    template_name = "pages/" + field.data
+    if not field.data:
```

Comment:
Do we still want this check even though we have `DataRequired()` validator invoked before?